test/cert-ceremonies/root-ceremony-rsa.yaml

```diff
@@ -20,3 +20,14 @@ certificate-profile:
     key-usages:
         - Cert Sign
         - CRL Sign
+skip-lints:
+   - e_ext_authority_key_identifier_missing
+   - e_ext_authority_key_identifier_no_key_identifier
+   - e_sub_ca_aia_does_not_contain_ocsp_url
```

Comment:
This one was removed by SC31, but we haven't updated our vendored version of zlint yet.